### ISSUE TITLE
File browser uses single click to select/navigate 

### ISSUE THREAD

**Issue by MBilderbeek** [openMSX/openMSX#1740]
This is not what people expect, as most native file browsers require a double click for that. So many people do that and end up in the wrong directory, leading to frustration.

Propose to modify the code for this.

**Comment by MBilderbeek** [comment]
@m9710797 suggested to try this:
```
diff --git a/src/3rdparty/ImGuiFileDialog/ImGuiFileDialog.cc b/src/3rdparty/ImGuiFileDialog/ImGuiFileDialog.cc
index 4278c14f3..8c9c44442 100644
--- a/src/3rdparty/ImGuiFileDialog/ImGuiFileDialog.cc
+++ b/src/3rdparty/ImGuiFileDialog/ImGuiFileDialog.cc
@@ -3815,7 +3815,7 @@ void IGFD::FileDialog::m_SelectableItem(int vidx, std::shared_ptr<FileInfos> vIn
                 {
                     fdi.SelectFileName(m_FileDialogInternal, vInfos);
                 } else {
-                    fdi.puPathClicked = fdi.SelectDirectory(vInfos);
+                    //fdi.puPathClicked = fdi.SelectDirectory(vInfos);
                 }
             } else  // no nav system => classic behavior
             {

```
looks like that gives the desired behaviour indeed. I can still select files and directories and have to double-click to change into a directory.

**Comment by MBilderbeek** [comment]
Should be fixed with aa1adbf15934a11071dbbdd03bbd6c035b035a8d
